blog/2024/09/juliacon-2025-announcement.md

```diff
@@ -0,0 +1,102 @@
+@def title = "JuliaCon Global 2025 and JuliaCon 2024 Wrap-Up"
+@def authors = "JuliaCon Organizing Committee"
+@def published = "18 September 2024"
+@def rss_pubdate = Date(2024, 9, 18)
+@def rss = """JuliaCon Global 2025 and JuliaCon 2024 Wrap-Up Blogpost."""
+
+![Pittsburgh Skyline](/assets/blog/2024-JuliaCon/AdobeStock_50245829.jpeg)
+#### We’re happy to announce that JuliaCon Global 2025 will be held in Pittsburgh, Pennsylvania, USA July 21 - July 26, 2025!
+
+A few important date estimates for you:
+
+- Call for Proposals will be open November 1, 2024 - January 15, 2025 with acceptances released March 1, 2025
+- Volunteer and scholarship applications will be open from December 1, 2024 - February 28, 2025
+- Early-bird tickets will go on sale late this year, full-price ticket sales will start on March 15, 2025
+
+We look forward to seeing you in Pittsburgh, and please keep an eye out for future announcements!
+
+**If you’re interested in joining the JuliaCon Global 2025 organizing committee** please fill out this form: <https://forms.gle/mYNPDgSvAYyxcNM6A>
+
+## Julia Community Efforts
+
+We have a couple new community efforts we’d like to talk about!
+
+### Julia Online Videos
+
+We have a lot of community members who want to give presentations at JuliaCon each year but can’t make it! We receive a lot of requests to submit videos to the conference, but for now we intend to run JuliaCon as an in-person conference first and foremost. Hybrid conferences take a lot of effort, and have a negative impact on the in-person experience.
+
+As an alternative, we are starting a new trial program called ***Julia Online*** for community members to propose videos for upload to the Julia YouTube Channel. If your proposal is accepted, you’ll be asked to prepare your proposal into a full video for upload! This call for proposals will be evergreen and will stay open for as long as we run this program, so you are welcome to propose a video at any time.
```

Comment:
`***Julia Online***` is rendered as is, maybe you wanted bold and/or italic?

Reply:
Oh yes, how are bold and italics handled in this version of MarkDown?

Reply:
I don't know for sure, but could try with
```suggestion
As an alternative, we are starting a new trial program called _**Julia Online**_ for community members to propose videos for upload to the Julia YouTube Channel. If your proposal is accepted, you’ll be asked to prepare your proposal into a full video for upload! This call for proposals will be evergreen and will stay open for as long as we run this program, so you are welcome to propose a video at any time.
```

Reply:
Looks like it worked: https://julialang.netlify.app/previews/pr2145/blog/2024/09/juliacon-2025-announcement/#julia_online_videos